CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 ### Added
 * localize_data within __main__.py added option to use/not use water mask for ionosphere processing
 * Added option to estimate burst phase jumps in ionosphere computation
+* Added packing of additional attributes for ionosphere computation into GUNW
```

Comment:
Can you please clarify what these attributes are here and in the PR?

Reply:
added to PR and changelog. Let me know if this is sufficient